gps_nmea_parser/src/include/gps/gps.h

```diff
@@ -115,6 +125,37 @@ extern "C" {
 #define GPS_CFG_STATEMENT_GPGSV_SAT_DET     0
 #endif
 
+/**
+ * \brief           Enables `1` or disables `0` parsing and generation
+ *                  of PUBX (uBlox) messages
+ *
+ *                  PUBX are a nonstandard ublox-specific extensions,
+ *                  so disabled by default.
+ */
+#ifndef GPS_CFG_STATEMENT_PUBX
+#define GPS_CFG_STATEMENT_PUBX     0
+#endif
+
+#if GPS_CFG_STATEMENT_PUBX
+
+/**
+ * \brief           Enables `1` or disables `0` parsing and generation
+ *                  of PUBX (uBlox) TIME messages.
+ *
+ * \note            TIME messages can be used to obtain:
+ *                      - UTC time of week
+ *                      - UTC week number
+ *                      - Leap seconds (allows conversion to eg. TAI)
+ *
+ *                  This is a nonstandard ublox-specific extension,
+ *                  so disabled by default.
+ */
+#ifndef GPS_CFG_STATEMENT_PUBX_TIME
+#define GPS_CFG_STATEMENT_PUBX_TIME     0
+#endif
+
+#endif /* GPS_CFG_STATEMENT_PUBX */
```

Comment:
Config doesn't need to have conditional check first. You can remove lines 139 and 157.

Reply:
ok, see 5ee9b180